twigshim.module

```diff
@@ -71,6 +71,12 @@ function twigshim_render($template, $variables = array()) {
       $templates = "$theme_path/templates";
     }
 
+    // Create a twig worker.
+    $loader = new Twig_Loader_Filesystem(DRUPAL_ROOT . base_path() . $templates);
+
+    // Add Drupal's web root so that we can provide absolute template paths.
```

Comment:
I'm still confused. Why do we want to support absolute template paths? If you meant your comment to say "_relative_ template paths" instead of "_absolute_ template paths," this code isn't needed, as paths relative to the Drupal root already work out of the box. This is the same as with your comment on Gremy's PR about the cache directory; these paths are relative to the web root by default.